packages/lookup/src/ux-lookup.ts

```diff
@@ -120,20 +120,29 @@ export class UxLookup implements UxComponent, EventListenerObject {
     this.debounceNumber = normalizeNumberAttribute(this.debounce);
   }
 
+  @bindable
+  preloadOptions: boolean | string;
+
   bind() {
     this.themeChanged(this.theme);
     this.valueFieldChanged();
     this.displayFieldChanged();
     this.optionsChanged();
   }
 
-  attached() {
+  async attached() {
     this.inputElement = this.element.parentElement?.querySelector<UxInputElement | HTMLInputElement>('ux-input,input');
     if (this.inputElement) {
       inputEvents.forEach(x => this.inputElement!.addEventListener(x, this));
     }
     lookupEvents.forEach(x => this.element.addEventListener(x, this));
     this.valueChanged();
+    if (!this.value && normalizeBooleanAttribute('preload-options', this.preloadOptions)) {
+      try {
+        await this.loadOptions();
```

Comment:
why do we need await here?

Reply:
No particular reason other than being async all the through

Reply:
I see. Maybe let's avoid it if we don't need it. Having await in `attached` and await here generates bloated code as well.

Reply:
removed